release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -196,6 +196,27 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-storage"]
+=== Storage
+
+[id="ocp-4-6-csi-driver-cso"]
+==== CSI drivers now managed by the Cluster Storage Operator
+The Container Storage Interface (CSI) drivers for AWS Elastic Block Store (EBS), Red Hat Virtualization (oVirt), and OpenStack Manila shared file system service are now managed by the Cluster Storage Operator in the {product-title} back end.
+
+For AWS EBS and oVirt, this feature installs the CSI Driver Operator and driver in the `openshift-cluster-csi-drivers` namespace by default. For Manila, the CSI Operator is installed in `openshift-cluster-csi-drivers` and the driver is installed in the `openshift-manila-csi-driver` namespace.
+
+[IMPORTANT]
+====
+If you installed the AWS EBS Operator and driver on an {product-title} 4.5 cluster, you must uninstall them before you update to a newer version of {product-title}.
```

Comment:
We should mention that Manila CSI driver operator is no longer available in OLM and it's silently converted under CVO.

Reply:
Sure thing, @jsafrane. Does this look good now? I had to tweak the wording a bit to match the needs of both driver operators.